CHANGELOG.md

```diff
@@ -20,6 +20,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - `coreax.inverses.RegularisedInverseApproximator` replaced by `coreax.least_squares.RegularisedLeastSquaresSolver`
   - `coreax.inverses.LeastSquaresApproximator` replaced by `coreax.least_squares.MinimalEuclideanNormSolver`
   - `coreax.inverses.RandomisedEigendecompositionApproximator` replaced by `coreax.least_squares.RandomisedEigendecompositionSolver`
+- Refactoring of `coreax.kernel.py` into `coreax.kernels` sub-package:
+  - `kernels.util.py` holds utility functions relating to kernels e.g. `median_heuristic`.
```

Comment:
Mention that `block_data_convert` is now public.

Reply:
Will make this a private function again